### DISCUSSION TITLE
installing with docker to include extensions

### DISCUSSION THREAD

**Discussion by ghost** [ckan/ckan#7319] [Q&A]
hello,  I have built a CKAN instance on Ubuntu with several extensions and this is running on local host happily. 

due to my firm wanting this in docker and I correct in thinking the process is: 
1) install ckan from the site. 
2) add my extension packages. 
3) run docker compose to build the images.

**Comment by ghost** [comment]
also.  

I am trying to add 2 extensions: 

CKAN-Scheming,   which requires additional configuration, but if I add the code to my .env file it isnt being picked up and added to the ini file - not sure why. 

CKAN-Comments.  this requires a DB upgrade but unsure how to do this with Docker.

**Comment by ghost** [comment]
anyone?

**Comment by kowh-ai** [comment]
> hello, I have built a CKAN instance on Ubuntu with several extensions and this is running on local host happily.
> 
> due to my firm wanting this in docker and I correct in thinking the process is:
> 
> 1. install ckan from the site.
No. CKAN (and the supporting components) is installed as Docker images and run as Docker containers
> 2. add my extension packages.
Extensions can be installed into the running CKAN container or (better still) installed after updates to the CKAN Dockerfile (see: https://github.com/ckan/ckan-docker/blob/master/ckan/Dockerfile.dev)
> 3. run docker compose to build the images.
no need for this step as it has already been done

**Reply by ghost** [reply]
Thanks Brett, appreciate the help.
